docs/components/navigation.tsx

```diff
@@ -14,6 +14,17 @@ export default class Navigation extends React.Component<IProps> {
     return <div className={styles.navigation}>{this.renderTrees(this.props.tree.children)}</div>;
   }
   private renderTrees(trees: IFileTree[], depth = 0) {
+    // If there's a priority, order by priority. If not, order alphabetically.
+    trees.sort((a: IFileTree, b: IFileTree) =>
+      a.attributes.priority == null || b.attributes.priority == null
```

Comment:
This should be &&, not || I think.

Reply:
I don't think so, as || doesn't sort by priority unless all items in that division have a priority specified, which doesn't let JS fall through to the null comparison against 0. I'm actually checking the logical equivalent of !a && !b, while putting the ternary in the opposite order, if that makes sense.

Reply:
If either priority is null, then this compares the titles.
Should something with a set priority not always go infront of something without a priority?

Reply:
Ah I see what you want. Have done that, needed another layer of logic for the case where one is null but not the other, as null defaults to priority of 0.